<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsCalibration</class>
 <widget class="QWidget" name="OpsCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>351</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="l_29">
            <property name="text">
             <string>Pulse:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="kcfg_AutoModeIterations">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="kcfg_CalibrationPulseDuration">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Iterations:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_TwoAxisEnabled">
          <property name="text">
           <string>Two axis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_GuideAutoSquareSizeEnabled">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Automatically select the square size based on the selected star width.</string>
          </property>
          <property name="text">
           <string>Auto Square Size</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Reticle</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="l_26">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinBox_ReticleX">
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="l_27">
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinBox_ReticleY">
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Calibrated Values</string>
     </property>
     <widget class="QLabel" name="racal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RA</string>
      </property>
     </widget>
     <widget class="QLabel" name="ra_cal_degrees">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>degrees</string>
      </property>
     </widget>
     <widget class="QLabel" name="ra_cal_mspp">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ms/arcsec</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DEC</string>
      </property>
     </widget>
     <widget class="QLabel" name="dec_cal_degrees">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="dec_cal_degrees_unit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>degrees (swapped)</string>
      </property>
     </widget>
     <widget class="QLabel" name="dec_cal_mspp">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ms/arcsec</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_GuideCalibrationBacklash">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove DEC backlash when calibrating guider.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Remove DEC backlash in guide calibration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_ResetGuideCalibration">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When guiding is started, always clear calibration. Turning off this option can lead to unpredictable guiding failures.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Reset Guide Calibration After Each Mount Slew</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_ReuseGuideCalibration">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Store and attempt to reuse calibrations. This only works on opposite sides of the meridian if the pier-side can be determined. Option above, reset-on-slew, must be unchecked for this to be attempted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Store and reuse guide calibration when possible.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_ReverseDecOnPierSideChange">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When re-using a calibration, reverse the DEC angle if the current pier side differs from the one at calibration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Reverse DEC on pier-side change when reusing calibration.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
